source/hassio/installation.markdown

```diff
@@ -117,6 +117,19 @@ Optional:
 
  - `network-manager`
 
+<div class='note'>
+   The default installation of `network-manager` will set a random Mac address over wifi on each restart. This will be troublesome in any situation where you would like to set a static IP. Edit `/etc/NetworkManager/NetworkManager.conf` and configure as follows:
```

Comment:
You need to add an empty newline between the HTML & markdown to allow the parser to switch context.